src/Raven.Studio/typescript/viewmodels/database/settings/documentsCompression.ts

```diff
@@ -41,6 +43,8 @@ class documentsCompression extends viewModelBase {
         this.canAddAllCollections = ko.pureComputed(() => {
            return !!_.difference(this.allExistingCollections(), this.collectionsToCompress()).length;
         });
+        
+        this.storageReportUrl = ko.pureComputed(() => appUrl.forStatusStorageReport(this.activeDatabase().name));
```

Comment:
please check if this doesn't break anything when you change database via top switcher. Then active database might be null for a moment?

Reply:
done